### PR TITLE
feat(as-2831): add clamav to appeals service

### PR DESCRIPTION
**Pull request by rcmsoftwaredevelopment**
add clamav to appeals service

## Ticket Number
https://pins-ds.atlassian.net/browse/AS-2831

## Description of change
Add clamav to appeals service

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -301,3 +303,8 @@ services:
       DISABLE_UNOCONV: '1'
     ports:
       - 4006:4000
+
+  clamav-server:
+    image: mkodockx/docker-clamav
+    ports:
+      - 3310:3310
```
Comment (a6software) [inline docker-compose.yml:310]: this is going to need the equivalent helm chart / k8s setup I guess?

**Comment by a6software** [comment]
Seems good to me, though would strongly advise a devops review on that k8s config

**Comment by Russ-Clarke** [comment]
That K8s config is a bit beyond mine and Gareth's current knowledge. @JoshHarrisplanninginspectorate , would you be able to advise, please?

File: docker-compose.yml (hunk 1 above)
Comment (rcmsoftwaredevelopment) [reply docker-compose.yml:310]: Thank you for the reminder - already done and pushed.

**Review by Russ-Clarke — DISMISSED** [review]
Ray advised that Josh has checked this, so I'm happy to approve.

**Review by Russ-Clarke — APPROVED** [review]

**Pull request closed without merging**